src/components/AnimalCard/styles.ts

```diff
@@ -0,0 +1,54 @@
+import { ImageProps, TextProps, ViewProps } from 'react-native';
+import styled from 'styled-components/native';
+import { Theme } from '../../constants';
+
+interface CardHeaderProps extends ViewProps {
+    backgroundColor: string;
+}
+
+export const CardBox = styled.View<ViewProps>`
+    width: 344px;
+    background: ${Theme.default.background};
+    elevation: 3;
```

Comment:
A little irrelevant, but should we also put styling for IOS shadows? Elevation only works on Android...

Reply:
What is IOS?

Reply:
Ok, point taken.